src/libsyntax/tokenstream.rs

```diff
@@ -196,6 +201,10 @@ impl TokenStream {
         }
     }
 
+    pub fn builder() -> TokenStreamBuilder {
```

Comment:
I'll change to `TokenStreamBuilder::new()` (motivation for `TokenStream::builder()` was to avoid making people `use` anything else into their scopes).